src/current/cockroachcloud/connect-to-a-serverless-cluster.md

```diff
@@ -25,6 +25,18 @@ Removing or adding an authorized network on your {{ site.data.products.serverles
 
 {% include cockroachcloud/authorize-your-clusters-networks.md %}
 
+### Establish AWS PrivateLink
+
+Amazon Web Services (AWS) PrivateLink support allows customers to establish SQL access to their clusters entirely through private AWS infrastructure, without exposure to the public internet, affording enhanced security and performance.
+
+AWS PrivateLink is available only for multiregion {{ site.data.products.serverless }} clusters deployed on AWS. GCP Peering is not yet available for {{ site.data.products.serverless }}.
+
+To configure PrivateLink, you create the AWS PrivateLink connection in your or AWS console, then configure your cluster to allow connections from the connection. For more information and detailed instructions, refer to[Network Authorization for {{ site.data.products.dedicated }} clusters: AWS PrivateLink](network-authorization.html#aws-privatelink).
```

Comment:
I think we could leave out both of these sentences or maybe just remove the Dedicated part